product_configurator/models/product_config.py

```diff
@@ -685,7 +685,7 @@ def write(self, vals):
         try:
             self.validate_configuration(final=False)
         except ValidationError as exc:
-            raise ValidationError(_("%s") % exc.name) from exc
+            raise ValidationError(_("{}".format(exc)))
```

Comment:
Maybe `raise ValidationError(_("{}").format(exc))`?
Or even better, I can't see the point for this `except`, what if we remove it?